package com.example.HibernateDemo.model;

import jakarta.persistence.Embeddable;

@Embeddable
public class Address {

    private String houseNo;
    private String street;
    private String city;
    private String state;
    private String pincode;

    public String getHouseNo() {
        return houseNo;
    }

    public void setHouseNo(String houseNo) {
        this.houseNo = houseNo;
    }

    public String getStreet() {
        return street;
    }

    public void setStreet(String street) {
        this.street = street;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getPincode() {
        return pincode;
    }

    public void setPincode(String pincode) {
        this.pincode = pincode;
    }

    public Address(String houseNo, String pincode, String state, String street, String city) {
        this.houseNo = houseNo;
        this.pincode = pincode;
        this.state = state;
        this.street = street;
        this.city = city;
    }

    public Address(){};
}
